src/geant4/app/exampleN01.cc

```diff
@@ -0,0 +1,117 @@
+//
+//
+// --------------------------------------------------------------
+//      GEANT 4 - exampleN01
+// --------------------------------------------------------------
+
+#include "G4RunManager.hh"
+#include "G4UImanager.hh"
+#include "G4Timer.hh"
+
+#include "ExN01DetectorConstruction.hh"
+#include "ExN01PhysicsList.hh"
+#include "ExN01PrimaryGeneratorAction.hh"
+#include "ExN01ActionInitialization.hh"
+
+#include "ExN01UserScoreWriter.hh"
+
+#include "G4PhysListFactory.hh"
+#include "G4ScoringManager.hh"
+
+#ifdef G4VIS_USE
+#include "G4VisExecutive.hh"
+#endif
+
+#ifdef G4UI_USE
+#include "G4UIExecutive.hh"
+#endif
+
+#ifndef uwuw_hpp
+#define uwuw_hpp 1
+#include "uwuw.hpp"
+#endif
+
+int main(int argc, char* argv[]) {
+  G4Timer Timer;
+  Timer.Start();
+
+  // Construct the default run manager
+  G4RunManager* runManager = new G4RunManager;
+
+  // Activate command-based scorer
+  G4ScoringManager* scManager = G4ScoringManager::GetScoringManager();
+  scManager->SetVerboseLevel(1);
+  scManager->SetScoreWriter(new ExN01UserScoreWriter());
+
+  std::string uwuw_file(argv[1]); // file containing data & uwuw
+
+  // Activate UI-command base scorer
+  // load the UWUW data
+  UWUW* workflow_data = new UWUW(uwuw_file);
+
+  // setup detectors and scores
+  runManager->SetUserInitialization(new ExN01DetectorConstruction(workflow_data));
+
+  G4PhysListFactory* physListFactory = new G4PhysListFactory();
+  G4VUserPhysicsList* physicsList =
+      physListFactory->GetReferencePhysList("QGSP_BIC_HP");
+  runManager->SetUserInitialization(physicsList);
+
+
+  // set mandatory user action class
+  //
+  ExN01ActionInitialization* actionInitialization = new ExN01ActionInitialization(workflow_data);
+  runManager->SetUserInitialization(actionInitialization);
+
+  //  G4VUserPrimaryGeneratorAction* gen_action = new ExN01PrimaryGeneratorAction;
```

Comment:
Readability: Remove commented out code, here and throughout the file.

Reply:
I was actually thinking about keeping those, as this is an example file ?
any thought @makeclean @gonuke @pshriwise ?

Reply:
I guess I'd lean toward removing them unless there are comments added explaining why they're there for the purposes of the example. Other comments in this file seem to be explaining what each step in the example is doing, but here it seems like these were commented to ensure the example works.

Reply:
I think this file is modified from a version that used to be distributed with Geant4, so it might be instructive to leave them in.  However, this file is no longer distributed with Geant4, so probably not continued value....  Maybe better to create a new example based on a new standard Geant4  example, but that's out of scope for here.  I would also require comments describing why these lines would be commented out in a DAGMC version

Reply:
maybe this is particularly poor timing (and poor naming) but the example01.cc (and everything else present) really feature as a complete DAG-Geant4 program, linking with uwuw for materials (and tallies in some cases). There is likely some value in refactor, to factor out DagMaterial.cc and DagTally.cc (IIRC) as libraries for handling G4 materials and G4 tally objects.